platform/cluster/aws/templates/configure-vm-aws.sh

```diff
@@ -131,5 +131,11 @@ function run-user-script() {
   # AWS doesn't have as rich a metadata service as GCE does
   # Maybe specify an env var that is the path to a script?
   :
+  # Argo hack:
+  # Sometimes docker is not restarted by either cloud-final or health checker due to race.
+  # Best way is to set up correct systemd dependencies. For now, we just always force a restart after salt.
+  # This is not to run AWS meta user script. But it's at the right time after salt call.
+  echo "Force restarting docker daemon"
+  service docker restart
```

Comment:
I would rather add this to docker-healthcheck, which is a level-triggered enforcement (i.e. we keep on asserting state and change it to desired one)